downstream/modules/platform/ref-controller-system-requirements.adoc

```diff
@@ -34,8 +34,8 @@ h| Requirement | Required
 | *CPUs* | 4
 | *Local disk* a|
 * 40GB minimum with at least 20GB available under /var/lib/awx
-* Storage volume should be rated for a minimum baseline of 1500 IOPS
-* Projects are stored on control and hybrid, and for the duration of jobs, also on execution nodes. If the cluster has many large projects, consider doubling the GB in /var/lib/awx/projects, to avoid disk space errors
+* Storage volume must be rated for a minimum baseline of 1500 IOPS
+* Projects are stored on control and hybrid, and for the duration of jobs, are also stored on execution nodes. If the cluster has many large projects, consider doubling the GB in /var/lib/awx/projects, to avoid disk space errors
```

Comment:
control and hybrid, -> control and hybrid nodes,

Reply:
Fixed